src/test/java/org/openrewrite/java/testing/junit5/AddParameterizedTestAnnotationTest.java

```diff
@@ -102,6 +135,35 @@ void testIsOdd(int number) {
                   }
               }
               """
+          ),
+          //language=kotlin
+          kotlin(
+            """
+              import org.junit.jupiter.api.Test
+              import org.junit.jupiter.params.provider.ValueSource
+              import org.junit.jupiter.api.Assertions.assertTrue
+
+              class NumbersTest {
+                  @Test
+                  @ValueSource(ints = [1, 3, 5, -3, 15, Int.MAX_VALUE])
+                  fun testIsOdd(number: Int) {
+                      assertTrue(number % 2 != 0)
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.params.ParameterizedTest
+              import org.junit.jupiter.params.provider.ValueSource
+              import org.junit.jupiter.api.Assertions.assertTrue
+
+              class NumbersTest {
+                  @ParameterizedTest
+                  @ValueSource(ints = [1, 3, 5, -3, 15, Int.MAX_VALUE])
+                  fun testIsOdd(number: Int) {
+                      assertTrue(number % 2 != 0)
+                  }
+              }
+              """
```

Comment:
I appreciate how thorough you've been to add all these Kotlin test blocks @amitojduggal , although I think going forward we'd only need to prove Kotlin works with a single test, perhaps the `@DocumentExample`, without a need to duplicate the assertions for every test scenario. Then later we can only add explicit Kotlin tests where we see issues, or want different handling as compared to the Java implementation. That would better allow us to see where there might be something of note by having a Kotlin test, as opposed to having them for all tests. What are your thoughts there?

Reply:
Thanks @timtebeek , it makes a lot of sense, and it would help us speed it up. I can make these changes for both of the test classes.